lib/util/tooltip.coffee

```diff
@@ -31,9 +31,13 @@ class Tooltip
     @showOnHover()
     this
 
+  hide_: ->
```

Comment:
The main issue is that when the progress bars end you get an empty tooltip instantly, which then fades out. It would be nice if we could animate the size changes of the tooltip but I'm not sure that's easy to do.